openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -82,7 +84,8 @@ def create_account(self, username, password, email):
 
 @skip_unless_lms
 @ddt.ddt
-@patch('common.djangoapps.student.views.management.render_to_response', Mock(side_effect=mock_render_to_response, autospec=True))  # lint-amnesty, pylint: disable=line-too-long
+@patch('common.djangoapps.student.views.management.render_to_response',
+       Mock(side_effect=mock_render_to_response, autospec=True))  # lint-amnesty, pylint: disable=line-too-long
```

Comment:
```suggestion
       Mock(side_effect=mock_render_to_response, autospec=True))  # lint-amnesty
```

tiniest, unimportant thing, but I think since this line wrapped (yay!) you don't need the pylint  directive anymore. (I assume your editor was set to re-lint on save, and it's not a big deal  if you don't  make this change.)